fog-core.gemspec

```diff
@@ -2,6 +2,7 @@
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "fog/core/version"
+require 'english'
```

Comment:
@icco Sorry about that. I was just trying to get rid of the warning that `$INPUT_RECORD_SEPARATOR` isn't defined. I'll back that out.